server/codecs.test.ts

```diff
@@ -9,6 +9,9 @@ import { highlighterPromise } from "./server.ts"
 await highlighterPromise
 
 for (const runtime of Object.keys(testClients)) {
+  if (runtime === "polkadot") {
```

Comment:
Why?

Reply:
That codec test is failing. About to push a fix attempt.